src/benchmarks/micro/libraries/System.Runtime/Perf.Boolean.cs

```diff
@@ -12,8 +12,19 @@ namespace System.Tests
     [BenchmarkCategory(Categories.Libraries)]
     public class Perf_Boolean
     {
-        public static IEnumerable<object> StringValues => Values.Select(value => value.ToString()).ToArray();
-
+        public static IEnumerable<object> StringValues => new string[]{
+            "true", // strings, not bools, as otherwise strings in the logs have casing "True"
```

Comment:
I don't understand the comment. Why would the inputs to parsing be Booleans? Comment seems unnecessary.